src/main/java/gregtech/api/unification/material/materials/OrganicChemistryMaterials.java

```diff
@@ -17,9 +17,10 @@ public static void register() {
                 .polymer()
                 .color(0xDCDCDC)
                 .flags(GENERATE_GEAR, GENERATE_RING, GENERATE_FOIL)
-                .components(Carbon, 2, Hydrogen, 6, Oxygen, 1, Silicon, 1)
+                .components(Carbon, 6, Hydrogen, 18, Oxygen, 1, Silicon, 2)
```

Comment:
if you want 3 monomers, shouldn't it be 6 carbons, 18 hydrogens, 3 oxygens, and 3 silicon?